litestar/dto/factory/_backends/utils/renaming.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, cast
+
+if TYPE_CHECKING:
+    from litestar.dto.factory import DTOConfig
+    from litestar.dto.factory.data_structures import FieldDefinition
+    from litestar.dto.factory.types import RenameStrategy
+
+
+__all__ = ("determine_serialization_name",)
+
+
+class RenameStrategies:
+    """Useful renaming strategies than be used with :class:`DTOConfig`"""
+
+    def __init__(self, renaming_strategy: RenameStrategy) -> None:
+        self.renaming_strategy = renaming_strategy
+
+    def __call__(self, field_name: str) -> str:
+        if not isinstance(self.renaming_strategy, str):
+            return self.renaming_strategy(field_name)
+
+        return cast(str, getattr(self, self.renaming_strategy)(field_name))
+
+    @staticmethod
+    def upper(field_name: str) -> str:
+        return field_name.upper()
+
+    @staticmethod
+    def lower(field_name: str) -> str:
+        return field_name.lower()
+
+    @staticmethod
+    def camel(field_name: str) -> str:
+        return RenameStrategies._camelize(field_name)
+
+    @staticmethod
+    def pascal(field_name: str) -> str:
+        return RenameStrategies._camelize(field_name, capitalize_first_letter=True)
+
+    @staticmethod
+    def _camelize(string: str, capitalize_first_letter: bool = False) -> str:
```

Comment:
please move this out of the class, same for the other ones. 

also - we can simply use inflection. I would assume it will be inside our dependency chain in any case since its a hugely popular package (check the lock file maybe?), and if so we can simply add it as an explicit dependency.